go.mod

```diff
@@ -15,6 +15,6 @@ require (
 	github.com/kr/text v0.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	golang.org/x/mod v0.12.0 // indirect
-	golang.org/x/sys v0.10.0 // indirect
+	golang.org/x/sys v0.12.0 // indirect
```

Comment:
Is this and the below change to `go.sum` related to this PR?

Reply:
Sorry this somehow got into this, it's a leftover from an experiment. Reverted 😃